### PR TITLE
gdsgdsgs

### LINKED ISSUE #26885: gdsgdsgs
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

gdsgds

### Expected behavior

gsdg

### Steps to reproduce

gdsgdsg

### PrestaShop version(s) where the bug happened

8.0.0.0

### PHP version(s) where the bug happened

7.3

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by jf-viguier**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | gdsgdsgs
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/26885
| How to test?      | gdsgdsgs
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26900)

### CODE REVIEW

**Review by atomiix — APPROVED** [review]
LGTM! 😂

**Review by jolelievre — CHANGES_REQUESTED** [review]
Sorry but even though this change is legitimate I can't accept it without updating the specs first :trollface:

File: admin-dev/index.php [hunk 1]
```diff
@@ -92,3 +92,5 @@
     // Prepare and trigger LEGACY admin dispatcher
     Dispatcher::getInstance()->dispatch();
 }
+
+// gdsgdsgs
```
Comment (NeOMakinG) [inline admin-dev/index.php:96]:
```suggestion
// gdsgdsgs
```

Could you please translate your comment to Russian, we only use this lang in order to comment

Comment (jf-viguier) [reply admin-dev/index.php:96]: Nice move @NeOMakinG

**Review by matks — APPROVED** [review]
Good contribution

**Comment by NeOMakinG** [comment]
Let's write an article on `build` in order to communicate on this wonderful contribution

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
I'm sorry @jf-viguier but I can't agree with your change.
`gdsgdsgs` can be misunderstood by people and lead to a riot, especially in France.
I'm also afraid of side effects, when you insert data in `index.php` file, it will probably break all cart rules :thinking:

**Comment by NeOMakinG** [comment]
> I'm sorry @jf-viguier but I can't agree with your change. `gdsgdsgs` can be misunderstood by people and lead to a riot, especially in France. I'm also afraid of side effects, when you insert data in `index.php` file, it will probably break all cart rules 🤔

Can we disable this bot please? He is not useful anymore

**Comment by PierreRambaud** [comment]
> > I'm sorry @jf-viguier but I can't agree with your change. `gdsgdsgs` can be misunderstood by people and lead to a riot, especially in France. I'm also afraid of side effects, when you insert data in `index.php` file, it will probably break all cart rules thinking
> 
> Can we disable this bot please? He is not useful anymore


Alt Text

**Comment by jf-viguier** [comment]
@PierreRambaud
>  "break all cart rules"

This is the way

**Comment by SebSept** [comment]
Will you backport to 1.6 branch ?

**Comment by matks** [comment]
It was funny but I think we have to resume regular open source work 😆

**Pull request closed without merging**